ansible/README.md

```diff
@@ -1,7 +1,7 @@
 # Ansible MicroShift
 
 The purpose of this Ansible automation is to help gain insight into the footprint and start times of MicroShift.
-At this stage, we are capturing the first start, the second start (with cached images), disk usage data from a
+At present, we are capturing the first start, the second start (with cached images), disk usage data from a
```

Comment:
I have not seen this file. Could it be because we're collecting this information using vnstat command, which is not installed on the host by default? 
I created USHIFT-576 to address this issue.

Reply:
Yes, it's being collected with `vnstat`.

Reply:
vnstat seems to be installed when MicroShift host is configured, so it does not fall under devenv or R4H flows. We may need to install vnstat the same way we do for Prometheus.